tests/suite.bats

```diff
@@ -162,7 +162,7 @@ namespace: ${STACKER_NAMESPACE}
 stacks:
   - name: vpc
     class_path: stacker.tests.fixtures.mock_blueprints.VPC
-    stack_policy: tests/fixtures/stack_policies/default.json
+    stack_policy_path: fixtures/stack_policies/default.json
```

Comment:
We broke these up into individual files, so you'll probably want to merge master and get that cleared up here.